dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -69,7 +69,7 @@ class DALIGenericIterator(object):
                  for data or label correspondingly.
                  output_names should be distinct.
     size : int
-           Epoch size.
+          Number of samples in the epoch in the whole data set.
```

Comment:
What about something like
```suggestion
          Number of samples in the epoch (Usually the size of the dataset).
```

Reply:
Done